ci/images/alpine/Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM alpine
 
 RUN apk add --no-cache bash git cmake make gcc musl-dev pkgconfig \
     openssl-dev ncurses-dev libusb-dev eudev-dev json-c-dev libxml2-dev \
-    sqlite-dev libarchive-dev gettext-dev graphviz-dev dbus-dev
```

Comment:
I was doing my best not to include nodejs in our CI... Dark times have come.

Reply:
unfortunately the node is needed for gh actions